modules/ocf_hpc/files/add_slurm_users

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+'''
+Script that adds users in the 'ocfhpc' LDAP group to SLURM as a SLURM user.
+'''
+import subprocess
+
+from ocflib.account import search
+from ocflib.account import utils
+
+DEFAULT_ACCOUNT = 'users'
+
+
+def main():
+    ocfhpc_ldap_user_set = set(utils.list_group('ocfhpc'))
+
+    # This produces a newline-separated list of SLURM users.
+    ocfhpc_slurm_user_list = subprocess.check_output(
+        ('sacctmgr', '-noP', 'list', 'users', 'format=User'),
```

Comment:
Adding `AdminLevel=None` to this command will exclude unnecessary entries, like you and wporr.

Reply:
wporr and I are in the ocfhpc ldap group though, so it'll try to create users for us again in that case, and if we're set to a non-default account, it'll create a new user-account association.